config/admin.config.php

```diff
@@ -61,70 +59,70 @@
     'service_manager'               => [
         // Legacy Zend Framework aliases
         'aliases'   => [
-            DoctrineAutodiscoveryModel::class      => Model\DoctrineAutodiscoveryModel::class,
-            DoctrineMetadataServiceResource::class => Model\DoctrineMetadataServiceResource::class,
-            DoctrineRestServiceModelFactory::class => Model\DoctrineRestServiceModelFactory::class,
-            DoctrineRestServiceResource::class     => Model\DoctrineRestServiceResource::class,
-            DoctrineRpcServiceModelFactory::class  => Model\DoctrineRpcServiceModelFactory::class,
-            DoctrineRpcServiceResource::class      => Model\DoctrineRpcServiceResource::class,
+            'ZF\Apigility\Doctrine\Admin\Model\DoctrineAutodiscoveryModel'      => DoctrineAutodiscoveryModel::class,
+            'ZF\Apigility\Doctrine\Admin\Model\DoctrineMetadataServiceResource' => DoctrineMetadataServiceResource::class,
```

Comment:
Note the warnings from phpcs below. You can either (a) fix the issues, or (b) add annotations to suppress the warnings (the rule in question is General.Files.LineLength.TooLong).